specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+{
+  "swagger": "2.0",
```

Comment:
The folder of this file needs to be `stable`, not `ga`

Reply:
Done.